src/Components/Counter/Counter.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+function Counter({fruitCount, setFruitCount}) {
+
+    function sub() {
+        fruitCount > 0 ? setFruitCount(fruitCount - 1) : (setFruitCount(0));
```

Comment:
Als hier false uitkomt dan is de count al 0, dus nogmaals de 0 als value zetten is dan niet nodig

Reply:
Dank voor je comments! Ga het allemaal even uitproberen. Grt